server/services/participants.js

```diff
@@ -123,9 +123,11 @@ const createChangeHistory = (participantBody, changes) => {
     timestamp: new Date(),
     changes: [...changeDetails],
   };
-  newBody.history = participantBody.history
-    ? [newHistory, ...participantBody.history]
-    : [newHistory];
+  // handle histories that have bad data/aren't objects
```

Comment:
The previous code is perfectly alright.

Reply:
It is if prod doesn't have integer histories, otherwise this code needs changing doesn't it?